### PR TITLE
32 no traceback on failed

### PR DESCRIPTION
**Pull request by kryskool**
If attachment is incorrect, don't send traceback and proper return code (like 65 )
see http://www.postfix.org/pipe.8.html and https://gist.github.com/bojanrajkovic/831993#file-sysexits-h-L101

ref #32

### CODE REVIEW

File: modoboa_dmarc/lib.py [hunk 1]
```diff
@@ -139,12 +140,23 @@ def import_report_from_email(content):
...
         if part.get_content_type() not in ZIP_CONTENT_TYPES:
             continue
-        fpo = six.BytesIO(part.get_payload(decode=True))
-        import_archive(fpo, content_type=part.get_content_type())
-        fpo.close()
+        try:
+            fpo = six.BytesIO(part.get_payload(decode=True))
+            import_archive(fpo, content_type=part.get_content_type())
+        except OSError:
+            print('ERROR-PARSING')
```
Comment (tonioo) [inline modoboa_dmarc/lib.py]: Maybe we could add more detail about the error?

Comment (kryskool) [reply modoboa_dmarc/lib.py]: No problem, it's just a message for the test, i found another method

**Comment by kryskool** [comment]
Test on Python 2.7 does not retrun the same exception
Test on Python 3.5 fail because pip not found

**Comment by kryskool** [comment]
@tonioo it's done, thanks for the review

**Pull request merged**